rbe/install-istio-testing.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+cd "$(dirname "$0")"
+
+if [[ ! -f install.sh ]]; then
+  echo "ERROR: install.sh not found" >&1
+  echo "  curl https://raw.githubusercontent.com/kubernetes/test-infra/master/rbe/install.sh -o install.sh" >&1
+  exit 1
+fi
+
+if [[ ! -f configure.sh ]]; then
+  echo "ERROR: configure.sh not found" >&1
+  echo "  curl https://raw.githubusercontent.com/kubernetes/test-infra/master/rbe/configure.sh -o configure.sh" >&1
+  exit 1
+fi
+
+
+proj=istio-testing
+pool=prow-pool
```

Comment:
should this node be created first, or it is part of the install ?

Reply:
The script creates an rbe pool with this name